easybuild/easyconfigs/m/mrc/mrc-1.3.13-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,39 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/01
+easyblock = 'CMakeMake'
+
+name = 'mrc'
+version = '1.3.13'
+
+homepage = 'https://github.com/mhekkel/mrc'
+description = """Maartens Resource Compiler to store resources in the static section of an ELF binary."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+github_account = 'mhekkel'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['mrc-1.3.13_relax_libmcfp_version.patch']
+checksums = [
+    {'v1.3.13.tar.gz': 'cf93dba0d4fb6c5908a69927eac5738d0a8fa621886e8933c429618649db4d5e'},
+    {'mrc-1.3.13_relax_libmcfp_version.patch': '024ec3b03ef74d7a8716ad93e4ae240d5c5c53b0487910f48956bd6f311f57d4'},
+]
+
+
+configopts = '-DFETCHCONTENT_FULLY_DISCONNECTED=ON CMAKE_EXPORT_COMPILE_COMMANDS'
```

Comment:
That CMAKE_EXPORT_COMPILE_COMMANDS is missing -D and =xxx

But it is not really needed for anything anyway so unless I misunderstand something here it can be dropped from all places where you use it.

If we would ever want it we would make it an option to the CMakeMake easyblock instead of explicitly adding it to configopts.

Reply:
removed

Reply:
I meant that you should remove it from all the easyconfigs in this PR in case that was unclear.

Reply:
@akesandgren 
done.
btw, wouldn't it also make sense to have -DFETCHCONTENT_FULLY_DISCONNECTED=ON set by default in the CMakeMake easyblock?

Reply:
Probably, please open an issue about it so we have it on record.